autogen/oai/client.py

```diff
@@ -359,6 +359,22 @@ def get_usage(response: Union[ChatCompletion, Completion]) -> Dict:
         }
 
 
+class OpenAI_O1(OpenAIClient):
+
+    def __init__(self, **kwargs):
+        super().__init__(OpenAI(**kwargs))
+
+    def create(self, params: Dict[str, Any]) -> ChatCompletion:
+        print(params["messages"])
```

Comment:
o1 models do not support streaming, tooling and func calling also, in this case should we restrict the usage here ?